packages/alfa-dom/src/node/query/element-id-map.ts

```diff
@@ -0,0 +1,32 @@
+import { Cache } from "@siteimprove/alfa-cache";
+import { Map } from "@siteimprove/alfa-map";
+
+import { Query } from ".";
+import { Node } from "../../node";
+import { Document } from "../document";
+import { Element } from "../element";
+
+const elementMapCache = Cache.empty<Document, Map<string, Element>>();
+
+/**
+ * @public
+ */
```

Comment:
```suggestion
 * @remarks Since `id` are scoped to trees, and do not cross shadow or content boundaries,
 * we never need traversal options.
 */
```